tox.ini

```diff
@@ -6,8 +6,8 @@ passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
 deps =
     -r{toxinidir}/requirements-tests.txt
     lockfile
-    py{2.6,2.7}: paramiko
```

Comment:
I think we need to investigate the paramiko breakage instead of stop testing it. But having a broken CI is indeed bad.